### DISCUSSION TITLE
After creating a pop up window, how to set all other widgets to be non-interactive

### DISCUSSION THREAD

**Discussion by asingingbird** [emilk/egui#1509] [Q&A] [answered]
For instance, in the `file_dialog` example, when I clicked the `Open file` button to choose a file, I want the whole UI to become unclickable except for the file chooser dialog.

In fact, many apps do that, when a dialog pops up, the rest of the UI just becomes non-interactive, and if the user clicks on it, it blinks, bells and refuses to act on it.

Does egui have this feature now and how to do that ?

**Comment by asingingbird** [comment]
In the `file_dialog` example, after the `Open file` button was clicked, the `UI` did become kind of non-interactive because the `update` method was blocked on picking a file, yet it's still resizable and can be minimized, maximized and closed, and more importantly the UI was not acting correctly, for example the extend area turned black when resizing the window.

**Comment by emilk** [comment, marked as answer]
This is known as "modal" windows/dialogs, and has not yet been added to egui: https://github.com/emilk/egui/issues/686
